cmd/celestia-appd/cmd/root.go

```diff
@@ -36,6 +36,15 @@ const (
 
 	// TimeoutCommit is a flag that can be used to override the timeout_commit.
 	TimeoutCommitFlag = "timeout-commit"
+
+	// PyroscopeURLFlag is a flag to specify the URL for pyroscope profiling.
+	PyroscopeURLFlag = "pyroscope-url"
```

Comment:
There is already a pyroscope-url flag as reported in `celestia-appd start --help`:

```
      --pyroscope-url string                            URL of the Pyroscope instance to use for continuous profiling. If not specified, profiling will not be enabled
```

Before we add another flag, we should investigate why the existing flag doesn't work as expected.